gnome-shell/src/gnome-shell-sass/_common.scss

```diff
@@ -1216,12 +1230,11 @@ StScrollBar {
     &:hover, &:focus {
       background-color: $_hover_bg_color;
       color: $fg_color;
-      border: none;
-      padding: 14px;
     }
     &:active { 
       background-color: $_active_bg_color;
       color: $fg_color;
+      border-color: $selected_borders_color;
```

Comment:
Ah, no this makes sense for upstream but not for us. @3v1n0 Could you remove this line again?
They have blue system menu disks so this makes sense for them

Reply:
Ok, sorry for being late, but you know... Holidays :)

Anyways, fixed now.